foreshadow/tests/test_cleaners/test_data_cleaner.py

```diff
@@ -146,7 +146,23 @@ def test_numerical_input():
     cs = ColumnSharer()
     dc = DataCleaner(column_sharer=cs)
     dc.fit(data)
-    # transformed_data = dc.transform(data)
+    transformed_data = dc.transform(data)
+    assert np.array_equal(transformed_data, data)
+
+
+def test_numerical_input_fittransform():
+    """Test numerical input."""
+    import numpy as np
+    import pandas as pd
+    from foreshadow.cleaners import DataCleaner
+    from foreshadow.core.column_sharer import ColumnSharer
+
+    columns = ["financials"]
+    data = pd.DataFrame({"financials": np.arange(10)}, columns=columns)
+    cs = ColumnSharer()
+    dc = DataCleaner(column_sharer=cs)
+    transformed_data = dc.fit_transform(data)
```

Comment:
Might you be able to test them in the same test if they require the same setup (I know I know it breaks my cardinal rule but creating a fixture isn't worth the overhead)

Reply:
I'll look at it.